docs/providers/aws/guide/serverless.yml.md

```diff
@@ -269,6 +274,13 @@ functions:
             identitySource: method.request.header.Authorization
             identityValidationExpression: someRegex
             type: token # token or request. Determines input to the authorizer function, called with the auth token or the entire request event. Defaults to token
+          schema:
```

Comment:
We can remove `schema` from here now, as I would consider this as deprecated

Reply:
👍